samples/client/petstore/typescript-fetch/builds/allOf-nullable/Club.md

```diff
@@ -0,0 +1,32 @@
+
+# Club
+
+
+## Properties
+
+Name | Type
+------------ | -------------
+**owner** | [**Owner**](Owner.md)
+
+## Example
+
+```typescript
+import { Club } from ''
+
+// TODO: Update the object below with actual values
+const example: Club = {
+    "owner": null,
+}
+
+console.log(example)
+
+// Convert the instance to a JSON string
+const exampleJSON: string = JSON.stringify(example)
+console.log(exampleJSON)
+
+// Parse the JSON string back to an object
+const exampleParsed = JSON.parse(exampleJSON) as Club
+console.log(exampleParsed)
+```
+
+
```

Comment:
i wonder if you can manually remove these files (new md files at the top level of the output folder)

Reply:
Can I just delete the typescript-fetch folder and have it rebuilt?

Reply:
Deleted root level markdown files 👍